src/develop/masks/brush.c

```diff
@@ -2156,8 +2162,9 @@ static void dt_brush_events_post_expose(cairo_t *cr, float zoom_scale, dt_masks_
     }
     else
     {
-      float masks_border, masks_hardness, masks_density;
-      float radius, oldradius, opacity, oldopacity, pressure;
+      float masks_border = 0.0, masks_hardness = 0.0, masks_density = 0.0;
```

Comment:
All 0.0 should end with f : 0.0f

This will probably fix the CI builder.

Reply:
And indeed it did. Thanks!